docs/index.md

```diff
@@ -42,53 +40,50 @@ See [Building with Vanilla](/building-vanilla) and [Customising Vanilla](/custom
   </div>
 </div>
 
-<hr class="is-deep">
+<br>
 
-<div class="u-equal-height">
+<div class="row u-equal-height">
   <div class="col-12">
     <h3>Download</h3>
-    <p>Download the latest version of Vanilla from GitHub.</p>
+    <span>Download the latest version of Vanilla from GitHub.</span>
```

Comment:
This should be a P, it breaks spacing at other viewports:
![image](https://user-images.githubusercontent.com/118614/62929560-a1ecf480-bdb2-11e9-96cf-22d1ad4c2cfd.png)

Perhaps just put the button on the next line?

Reply:
Cool, I can fix that up 👍